Shared/Samples/Show extruded features/ShowExtrudedFeaturesView.swift

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct ShowExtrudedFeaturesView: View {
+    /// A scene with a topographic basemap centered on the US.
+    @State private var scene: ArcGIS.Scene = {
+        let scene = Scene(basemapStyle: .arcGISTopographic)
+        
+        // Set the scene view's viewpoint specified by the camera position.
+        let point = Point(x: -99.659448, y: 20.513652, z: 12_940_924, spatialReference: .wgs84)
+        let camera = Camera(lookingAt: point, distance: 0, heading: 0, pitch: 15, roll: 0)
+        scene.initialViewpoint = Viewpoint(boundingGeometry: point, camera: camera)
```

Comment:
Me personally don't like using a point as "bounding geometry" as it has no effect, and doesn't make sense grammatically. Instead, for setting a 3D viewpoint with a camera, use [Viewpoint.init(center:scale:rotation:camera:)](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/viewpoint/init(center:scale:rotation:camera:))